.codecov.yml

```diff
@@ -10,3 +10,4 @@ ignore:
   - "ccf/app/wsgi.py"
   - "ccf/app/__init__.py"
   - "ccf/app/settings/prod.py"
+  - "ccf/Docker Test Algorithm/invert.py"
```

Comment:
It's probably not good to have spaces in file or folder names...

Reply:
Addressed in 725f61d]